lib/traject/config/folio_config.rb

```diff
@@ -223,3 +225,11 @@ def holdings(record, context)
 to_field 'bound_with_parents_struct' do |record, accumulator|
   accumulator << record.bound_with_parents.to_json
 end
+
+##
+# Skip records for missing `item_display` field
+each_record do |record, context|
+  if context.output_hash['item_display'].blank? && context.output_hash['url_fulltext'].blank? && record.bound_with_parents.blank?
+    context.skip!('No item_display or url_fulltext field')
+  end
+end
```

Comment:
How's this work for on-order stuff? Doesn't that not have items?

Reply:
Some do, some don't. The ones that don't have items get a placeholder `item_display`:

https://github.com/sul-dlss/searchworks_traject_indexer/blob/main/lib/traject/config/sirsi_config.rb#L2398